brownie/network/contract.py

```diff
@@ -130,7 +131,11 @@ def decode_input(self, calldata: Union[str, bytes]) -> Tuple[str, Any]:
         if not isinstance(calldata, HexBytes):
             calldata = HexBytes(calldata)
 
-        fn_selector = calldata[:4].hex()  # type: ignore
+        if HEXBYTES_LT_1_0_0:
```

Comment:
must be a cleaner way here

```py
fn_selector = calldata[:4].hex()
if not fn_selector.startswith('0x'):
    fn_selector = f"0x{fn_selector}"
```

tho this is where you probably tell me that your method is a few milliseconds faster :rofl: 
```

Reply:
Ahh I see you're using `removeprefix` everywhere

```py
fn_selector = f"0x{calldata[:4].hex().removeprefix('0x')"
```

is that too many things in one line?  seems to address everything without having to import from another file just to verify a package version number.